docs/helm/helm_notes.md

```diff
@@ -0,0 +1,44 @@
+## helm notes
```

Comment:
Would you mind deleting this documentation? Doesn't look like it was cleaned up by `make docs`.

Reply:
Thank you for the review. I have updated the PR.